src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+
+type Props = {
+  isHidden: boolean;
+  setIsHidden: (arg: boolean) => void;
+  onError: string;
+};
+
+export const ErrorNotification: React.FC<Props> = (
+  {
+    isHidden,
+    setIsHidden,
+    onError,
+  },
+) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: isHidden },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
```

Comment:
Not critical, but I would name it like `className="button-hide"` or `button-close`